### PR TITLE
Implement `IndexFieldTypePollerAdapter` for ES7.

### PR DESCRIPTION
**Pull request by dennisoelkers**
## Motivation and Context

This PR is implementing the `IndexFieldTypePollerAdapter` for ES7, required to retrieve field types periodically.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (non-breaking change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: graylog-storage-elasticsearch7/src/main/java/org/graylog/storage/elasticsearch7/mapping/FieldMappingApi.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import javax.inject.Inject;
+import java.io.IOException;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class FieldMappingApi {
+    private final ObjectMapper objectMapper;
+
+    @Inject
+    public FieldMappingApi(ObjectMapper objectMapper) {
```
Comment (alex-konn) [inline graylog-storage-elasticsearch7/src/main/java/org/graylog/storage/elasticsearch7/mapping/FieldMappingApi.java:19]: Let's inject `RestHighLevelClient` in the constructor, as we do for the other `*Api` wrappers.

File: graylog-storage-elasticsearch6/src/main/java/org/graylog/storage/elasticsearch6/IndexFieldTypePollerAdapterES6.java [hunk 2]
```diff
@@ -51,7 +52,7 @@ public IndexFieldTypePollerAdapterES6(JestClient jestClient) {
         final JsonNode properties = result.getJsonObject()
                 .path(indexName)
                 .path("mappings")
-                .path("message") // TODO: Hardcoded index type name
+                .path(IndexMapping.TYPE_MESSAGE) // TODO: Hardcoded index type name
```
Comment (alex-konn) [inline graylog-storage-elasticsearch6/src/main/java/org/graylog/storage/elasticsearch6/IndexFieldTypePollerAdapterES6.java:55]: Uhm, that comment can go now, as it's no longer hard coded, is it?

Comment (dennisoelkers) [reply graylog-storage-elasticsearch6/src/main/java/org/graylog/storage/elasticsearch6/IndexFieldTypePollerAdapterES6.java:55]: :heavy_check_mark:

File: graylog-storage-elasticsearch7/src/main/java/org/graylog/storage/elasticsearch7/mapping/FieldMappingApi.java (hunk 1 above)
Comment (dennisoelkers) [reply graylog-storage-elasticsearch7/src/main/java/org/graylog/storage/elasticsearch7/mapping/FieldMappingApi.java]: :heavy_check_mark:

**Pull request merged**